### ISSUE TITLE
Fancy color change

### ISSUE THREAD

**Issue by manparvesh** [IdeaTrackerPlus/IdeaTrackerPlus#23]
Description
-------------------------------------------
Ripple effect while changing color of app. Something like [this](https://stackoverflow.com/questions/27872324/how-can-i-animate-the-color-change-of-the-statusbar-and-toolbar-like-the-new-ca)

**Comment by ArihantJain456** [comment]
hey I would like to do it.

**Comment by manparvesh** [comment]
@ArihantJain456 sure

**Comment by ArihantJain456** [comment]
Method use for creating such effect is `ViewAnimationUtils.createCircularReveal` which works only in lollipop and above versions.

**Comment by manparvesh** [comment]
@ArihantJain456 in that case, we can at least apply this to the versions that support it. Lollipop and the greater versions of Android are used by a really big percentage of phones.

**Comment by ArihantJain456** [comment]
also there are more issues ..

1. statusbar color resets every time we open app.
2. on back press app gets close even when right drawer is opened. For better user experience this  should fix

**Comment by manparvesh** [comment]
@ArihantJain456 thanks for pointing out! Feel free to add separate issues for anything you find is missing or wrong!

**Comment by ArihantJain456** [comment]
@manparvesh ripple effect would not be visible much if right drawer remains open on color change..so what should be done in this case?

**Comment by manparvesh** [comment]
@ArihantJain456 yeah you're right. But closing it after selecting a color would increase the work for the user. But maybe we can ignore that

**Comment by ArihantJain456** [comment]
sorry , but what to ignore user work or ripple effect
